### ISSUE TITLE
Allow player to spawn new objects

### ISSUE THREAD

**Issue by mohrtw** [xrchat/xrchat-client#54]
Select objects from our backend or external services (Sketchfab, TurboSquid, Google Poly, etc...)

**Comment by mohrtw** [comment]
Keep track of performance impact of new objects. Warn users when adding an asset will severely impact performance.

**Comment by shawticus** [comment]
I don't think we do a sketchfab thing here, I think we define objects inside our object API and those are the usables in the world only -- that way we have a lot of QC over what happens in the world.
